Libplanet.Explorer/Queries/VoteQuery.cs

```diff
@@ -0,0 +1,246 @@
+                    return CountVotes(desc, offset, limit);
+                }
+            );
+
+            Field<NonNullGraphType<VoteCountType>>(
+                "countValidatorVotes",
+                arguments: new QueryArguments(
+                    new QueryArgument<BooleanGraphType>
+                    {
+                        Name = "desc",
+                        DefaultValue = false,
+                    },
+                    new QueryArgument<IntGraphType>
+                    {
+                        Name = "offset",
+                        DefaultValue = 0,
+                    },
+                    new QueryArgument<IntGraphType>
+                    {
+                        Name = "limit",
+                    },
+                    new QueryArgument<PublicKeyType>
+                    {
+                        Name = "validator",
+                    }
+                ),
+                resolve: context =>
+                {
+                    bool desc = context.GetArgument<bool>("desc");
+                    long offset = context.GetArgument<long>("offset");
+                    int? limit = context.GetArgument<int?>("limit", null);
+                    PublicKey validator = context.GetArgument<PublicKey>("validator");
+                    return CountValidatorVotes(desc, offset, limit, validator);
+                }
+            );
+        }
+
+        private static IBlockChainContext<T> ChainContext { get; set; }
+
+        private static BlockChain<T> Chain => ChainContext.BlockChain;
+
+        internal IEnumerable<Vote> ListVotes(
+            bool desc,
+            long offset,
+            long? limit)
+        {
+            long tipIndex = Chain.Tip.Index;
+            IStore store = ChainContext.Store;
+
+            if (desc)
+            {
+                if (offset < 0)
+                {
+                    offset = tipIndex + offset + 1;
+                }
+                else
+                {
+                    offset = tipIndex - offset + 1 - (limit ?? 0);
+                }
+            }
+            else
+            {
+                if (offset < 0)
+                {
+                    offset = tipIndex + offset + 1;
+                }
+            }
+
+            var indexList = store.IterateIndexes(
+                    Chain.Id,
+                    (int)offset,
+                    limit == null ? null : (int)limit)
+                .Select((value, i) => new { i, value });
+
+            if (desc)
+            {
+                indexList = indexList.Reverse();
+            }
+
+            var result = new List<Vote>();
+            foreach (var index in indexList)
+            {
+                var votes = store.GetBlockCommit(index.value)?.Votes;
+                if (votes is { })
+                {
+                    result = result.Concat(store.GetBlockCommit(index.value).Votes).ToList();
```

Comment:
```suggestion
                if (store.GetBlockCommit(index.value)?.Votes is { } votes)
                {
                    result.AddRange(votes);
```

We already know `votes` is not `null`. 😗